ckanext/canada/templates/internal/recombinant/resource_edit.html

```diff
@@ -3,9 +3,19 @@
 {% import 'macros/canada_read.html' as cr %}
 
 {% block organization_display %}
+  {# this is a scoped macro call, passes in `org` dict #}
   {{- cr.split_bilingual_field(org.title, h.lang()) | truncate(60) -}}
 {% endblock %}
 
+{%- block subtitle_2 -%}
+  {{ cr.split_bilingual_field(organization.title, h.lang()) | safe }}
+{%- endblock -%}
+
+{% block adobe_analytics_creator %}
+  {% set _title_trans = cr.split_bilingual_field(organization.title, h.lang()) | safe %}
```

Comment:
`| safe`?

Reply:
Ah...I thought the safe filter did the escaping...but it omits it from getting escaped. That is NOT what I want ahaha

Reply:
Yeah, no. Our jinja2 is configured to escape everything *not* marked as safe when it appears as a variable in the output.